### PR TITLE
added euler's totient function in C

### PR DESCRIPTION
**Pull request by mharshita**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

I have added implementation of Euler's Totient Function in C language.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#445

### CODE REVIEW

**Comment by mharshita** [comment]
@atarax665  @plazzy99  I have added implementation of euler's totient function in C. Please review my PR and merge.

**Review by atarax665 — COMMENTED** [review]
Add in-line comments for better understanding.

File: C/math/Euler_totient_function.c [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    return 0;
+}
+
+/*
+SAMPLE INPUT:
+5
+
+SAMPLE OUTPUT:
+F(5) = 4
+*/
```
Comment (atarax665) [inline C/math/Euler_totient_function.c:46]: Add a blank line at the end.

**Comment by atarax665** [comment]
@mharshita Update README.md

**Review by siddharth25pandey — COMMENTED** [review]
@mharshita Copied from the [this](https://www.geeksforgeeks.org/eulers-totient-function/)
* Exactly same, only at some places variable names are changed.
* Don't copy & paste.
* Do it by your own.

**Comment by akanksha-saini** [comment]
Can I provide the solution in C for this issue? @siddharth25pandey

**Pull request closed without merging**